util/options.cc

```diff
@@ -1013,6 +1015,43 @@ parse_nameids (const char *name G_GNUC_UNUSED,
   return true;
 }
 
+static gboolean
+parse_drop_tables (const char *name G_GNUC_UNUSED,
+                   const char *arg,
+                   gpointer    data,
+                   GError    **error G_GNUC_UNUSED)
+{
+  subset_options_t *subset_opts = (subset_options_t *) data;
+
+  hb_set_clear (subset_opts->drop_tables);
+  char *s = (char *) arg;
+
+  while (s && *s)
+  {
+    while (*s && strchr (", ", *s))
```

Comment:
This loop (and the one below) can be replaced with calls to `strpbrk`.

Reply:
Switched to strtok